src/test/java/com/thebois/models/world/structures/TownHallTests.java

```diff
@@ -0,0 +1,53 @@
+package com.thebois.models.world.structures;
+
+import org.junit.jupiter.api.Test;
+
+import com.thebois.models.Position;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class TownHallTests {
+
+    @Test
+    public void TownHallIsAlwaysCompleted() {
```

Comment:
style

In Java, methods always begin with a lower case letter.

Reply:
Same goes for all methods in this class.

Reply:
Fixed it, idk what happened here